pkgs/servers/sql/postgresql/default.nix

```diff
@@ -15,7 +16,7 @@ let
     setOutputFlags = false; # $out retains configureFlags :-/
 
     buildInputs =
-      [ zlib readline openssl libxml2 makeWrapper ]
+      [ zlib readline openssl libxml2 makeWrapper systemd ]
```

Comment:
Shouldn't this be made optional for macos or is this already the case?

Reply:
You're right, will fix.